packages/lit-html/src/directives/live.ts

```diff
@@ -25,10 +25,7 @@ import {
   BOOLEAN_ATTRIBUTE_PART,
   ATTRIBUTE_PART,
 } from '../lit-html.js';
-
-// A sentinal value that can never appear as a part value except when set by
-// live(). Used to force a dirty-check to fail and cause a re-render.
-const RESET_VALUE = {};
+import {resetPartValue} from '../parts.js';
```

Comment:
What does this have to do with reserved properties?

Reply:
Added to PR description:
> * Adds `resetPartValue` helper for use by directives that were accessing the private `_value` property